src/model.js

```diff
@@ -424,11 +434,24 @@ const BookshelfModel = ModelBase.extend({
     let columnNames, candidates;
     if (_.isArray(arguments[1])) {
       columnNames = arguments[1];
-      candidates = _.drop(arguments, 2);
     } else {
       columnNames = null;
-      candidates = _.drop(arguments);
     }
```

Comment:
I think I'm not conveying my point. I'll give some examples with their results to see if we agree:

```js
morphTo('imageable', 'Site', ['Author', 'profile_pic'])
// columnNames: null
// morphValues: ['Site', 'sites'] ['Author', 'profile_pic' ]

morphTo('imageable, null, ['Site', 'cover_photo'], ['Author', 'profile_pic'])
// columnNames: null
// morphValues: ['Site', 'cover_photo'] ['Author', 'profile_pic' ]

morphTo('imageable, ['Site', 'cover_photo'], ['Author', 'profile_pic'])
// columnNames: ['Site', 'cover_photo']
// morphValues: ['Author', 'profile_pic' ]

morphTo('imageable, ['Site', 'cover_photo'], ['Author', 'profile_pic'])
// columnNames: ['Site', 'cover_photo']
// morphValues: ['Author', 'profile_pic' ]

morphTo('imageable, null, 'Site', ['Author', 'profile_pic'])
// columnNames: null
// morphValues: ['Site', 'sites'] ['Author', 'profile_pic' ]
```

What would you have expected differently from these or am I missing some case?